internal/application/filesystem.go

```diff
@@ -903,3 +929,21 @@ func getDNSHostName() string {
 	log.Infof("Chosen hostname : %s", dnsHostName)
 	return dnsHostName
 }
+
+//syncLoggingLevel matches log levels of Move2Kube-api and Move2Kube
+func syncLoggingLevel(loggingLevel, message string) {
+	switch {
```

Comment:
Also, we might want to add a default case here.

Reply:
Yes sure, thank you.